payjoin-cli/src/app.rs

```diff
@@ -208,12 +209,13 @@ impl App {
             .await??;
 
             println!("Sent fallback transaction");
-            let psbt = ctx.process_response(&mut response.into_reader())?;
-            if let Some(psbt) = psbt {
-                return Ok(psbt);
-            } else {
-                log::info!("No response yet for POST payjoin request, retrying some seconds");
-                std::thread::sleep(std::time::Duration::from_secs(5));
+            match ctx.process_response(&mut response.into_reader()) {
+                Ok(Some(psbt)) => return Ok(psbt),
+                Ok(None) => std::thread::sleep(std::time::Duration::from_secs(5)),
+                Err(re) => {
+                    println!("{}", re);
```

Comment:
extra log here

Reply:
why would we need both? aint the debug verbose enough?

Reply:
debug is verbose enough but in normal operation you the debug log is not printed to stdout. That only happens when debugging and manually setting RUST_LOG=debug